ci-operator/step-registry/storage/conf/storageclass/set-default-storageclass/storage-conf-storageclass-set-default-storageclass-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: storage-conf-storageclass-set-default-storageclass
+  from: cli
+  # inject oc binary
+  cli: latest
+  commands: storage-conf-storageclass-set-default-storageclass-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: EXPECTED_DEFAULT_STORAGECLASS
```

Comment:
Nit: `EXPECTED_DEFAULT_STORAGECLASS` sound like the script checks that the actual default SC matches the expected SC. `REQUIRED_DEFAULT_STORAGECLASS` / `NEW_DEFAULT_STORAGECLASS` would emphasize that the script modifies the default SC.

Reply:
Addressed, thks!